backend/src/modules/UsersRegistration/service/ValidateRegistration.js

```diff
@@ -0,0 +1,12 @@
+import joi from 'joi';
+
+export const registrationValidation = (data) => {
```

Comment:
W nazwach funkcji raczej oczekujemy schematu nazewnictwa tak jakbyśmy mówili np: "ustaw kolor tła" to nazwiamy"serBackgroundColor", a nie "backgroundColorSettings"

```suggestion
export const validateRegistration = (data) => {
```